pkgs/applications/misc/sioyek/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gumbo
+, harfbuzz
+, jbig2dec
+, mupdf
+, openjpeg
+, qt3d
+, qtbase
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sioyek";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "ahrm";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-UhZuELWfFfqj1jjCgZTD+X44i7jMpeDTbClkZarV2Zw=";
+  };
+
+  buildInputs = [ gumbo harfbuzz jbig2dec mupdf openjpeg qt3d qtbase ];
+
+  nativeBuildInputs = [ wrapQtAppsHook ];
+
+  buildPhase = ''
+    # Remove nonexistent lib and insert missing ones
+    sed -i 's/-lmupdf-threads/-lfreetype -lgumbo -ljbig2dec -lopenjp2 -ljpeg/' pdf_viewer_build_config.pro
+    qmake pdf_viewer_build_config.pro
+    make
+  '';
+
+  installPhase = ''
+    install -Dm755 sioyek $out/bin/sioyek
+    install -Dm644 tutorial.pdf $out/bin/tutorial.pdf
```

Comment:
maybe this belongs to $out/share/doc instead

Reply:
You mean `tutorial.pdf` right? I'm not too sure about the default search paths used by the program, but this is the first file it loads on a new installation. If we install it at a strange path it will launch with an error message on the first run.

Reply:
This file is the first file to be loaded on a fresh install. The search paths are a bit strange, I couldn't get it to find the file using anything else other than just `/bin/`.